content/en/observability_pipelines/guide/remap_reserved_attributes.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Remap Reserved Attributes
+disable_toc: false
+further_reading:
+- link: "observability_pipelines/processors/edit_fields/"
+  tag: "Documentation"
+  text: "Learn more about the Edit Fields processor"
+- link: "observability_pipelines/processors/custom_processor/"
+  tag: "Documentation"
+  text: "Learn more about the Custom Processor processor"
+---
+
+## Overview
+
+Observability Pipelines' processors enable you to add, edit, and remove log fields. Remapping attributes or rewriting values ensures your logs are processed and standardized properly. For the majority of processing use cases, use the Edit Fields processor to add, remap, or remove fields from your logs. For advanced use cases, use the Custom Processor to conditionally modify fields or rewrite a field's value.
+
+## What are reserved attributes and when they are used
```

Comment:
This heading is a little wordy. I might replace it with "About reserved attributes," but it's also fine if you prefer this sort of explicit style and have similar headings elsewhere in the OP docs.